README.md

```diff
@@ -114,6 +114,8 @@ docker build --tag gp-v2-migrations -f docker/Dockerfile.migration .
 docker run -ti -e FLYWAY_URL="jdbc:postgresql://host.docker.internal/?user="$USER"&password=" -v $PWD/database/sql:/flyway/sql gp-v2-migrations migrate
 ```
 
+In case you run into `java.net.UnknownHostException: host.docker.internal` add `--add-host=host.docker.internal:host-gateway` right after `docker run`.
+
 If you're combining a local postgres installation with docker flyway you have to add to the above `--network host` and change `host.docker.internal` to `localhost`.
```

Comment:
Are these two lines for solving the same issue?

Reply:
They should do different things. AFAIK `host.docker.internal` makes a container communicate with other containers whereas the second suggestions makes the container communicate with the host system. To be honest I didn't try running a psql data base on the host machine.
It seems on linux docker doesn't know what `host.docker.internal` is supposed to be so I had to add it myself.